PSReadLine/ViRegister.cs

```diff
@@ -0,0 +1,239 @@
+﻿using System;
+using System.Text;
+
+namespace Microsoft.PowerShell
+{
+    public partial class PSConsoleReadLine
+    {
+        /// <summary>
+        /// Used to report when the buffer is about to change
+        /// as part of a paste operation.
+        /// </summary>
+        public sealed class PasteEventArgs : EventArgs

```

Comment:
Does this really need to be public?

Reply:
It does not, I will change this to internal, or according to decisions made as per [this comment](https://github.com/PowerShell/PSReadLine/pull/811#discussion_r337091480).